grpc-interface-support/src/python.py

```diff
@@ -131,6 +199,76 @@ def __copy_code_and_templates(
             template_dir, self.__package_directory_path, files_to_copy, variables
         )
 
+    def __create_service_stub_source(self, service_name: str) -> None:
+        app_source_dir = os.path.join(get_workspace_dir(), "app", "src")
+
+        source_code = self.__service_grpc_code_extractor.create_source_stub_code()
+        variables = self.__create_stub_template_variables()
+        variables["service_source_code"] = "\n".join(source_code)
+
+        source_file_name = f"{service_name}ServiceStub.py"
+        source_file_path = os.path.join(app_source_dir, source_file_name)
+        templates.copy_templates(
+            get_template_dir(),
+            app_source_dir,
+            [templates.CopySpec(self.TEMPLATE_PATH, source_file_path)],
+            variables,
+        )
+
+    def __create_service_source(self, service_name: str) -> None:
+        app_source_dir = os.path.join(get_workspace_dir(), "app", "src")
+        service_source_file_name = f"{service_name}{self.TEMPLATE_PATH}"
+        service_source_file_path = os.path.join(
+            app_source_dir, service_source_file_name
+        )
+
+        if os.path.exists(service_source_file_path):
+            return
+
+        source_code = self.__service_grpc_code_extractor.create_source_stub_code()
+        source_code = self.__transform_service_source_code(source_code)
+
+        variables = self.__create_service_template_variables()
+        variables["service_source_code"] = "\n".join(source_code)
+
+        templates.copy_templates(
+            get_template_dir(),
+            app_source_dir,
+            [templates.CopySpec("ServiceImpl.py", service_source_file_path)],
+            variables,
+        )
+
+    def __transform_service_source_code(self, lines: List[str]) -> List[str]:
+        source_content: List[str] = replace_text_area(
+            lines,
+            '"""',
+            '"""',
+        )
+        source_content = replace_item_in_list(
+            source_content,
+            "context.set",
+            remove_empty=True,
+        )
+
+        return source_content
+
+    def __create_stub_template_variables(self) -> Dict[str, str]:
+        proto_file_prefix = self.__service_grpc_code_extractor.file_name_prefix
+        return {
+            "imports": f"import grpc{os.linesep}from {self.__service_name_lower}_service_sdk.{proto_file_prefix} import {self.__service_name}Servicer",
+            "service_name": self.__service_name,
+            "service_name_parent_postfix": "Servicer",
+            "service_name_postfix": "ServiceStub",
```

Comment:
We can decide to either have 2 template files or use the variables here because for the stub it is 

```
"service_name_parent_postfix": "Servicer",
"service_name_postfix": "ServiceStub",
```

but for the impl it is:

 "service_name_parent_postfix": "ServiceStub",
 "service_name_postfix": "",